service/src/test/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationServiceTest.java

```diff
@@ -187,4 +188,11 @@ private CaseData caseData(Representative... representatives) {
             .representatives(wrapElements(representatives))
             .build();
     }
+
+    private static List<String> buildDocumentsNamesList() {
+        List<String> documents = new ArrayList<>();
```

Comment:
java 11 has a handy util function here, if you want to construct a list of pre-determined objects then there is the `List.of` method, similar exists for maps and sets. Note that the returned list is immutable, so it won't work when the list is going to be updated